src/main/java/com/google/sps/servlets/GmailActionableEmailsServlet.java

```diff
@@ -113,7 +126,8 @@ public void doGet(
     List<ActionableMessage> actionableEmails =
         gmailClient.getActionableEmails(subjectLinePhrases, unreadOnly, nDays, METADATA_HEADERS)
             .stream()
-            .map(this::createActionableMessage)
+            .map((message) -> createActionableMessage(message, userEmail))
+            .sorted(Comparator.reverseOrder())
```

Comment:
Nice use of `sorted` in stream processing!

Reply:
Thank you!